backend/ccm/canvas_api/canvas_credential_manager.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from http import HTTPStatus
```

Comment:
Perhaps it's minor but we might want to look into having some code quality checks installed in this app as it gets developed. Running [isort ](https://pycqa.github.io/isort/) on this would clean these imports up a bit and make it more consistent. 

I've liked the [requirements of  Canvas API](https://github.com/ucfopen/canvasapi/blob/develop/scripts/run_tests.sh) and with you adding tests we could run coverage, black, isort, flake8 and _maybe_ some other custom scripts. Easier to apply this now than later. We could discuss this and create this as a new issue.